waves/.gitignore

```diff
@@ -22,3 +22,5 @@ yarn-error.log*
 .eslintcache
 
 src/wallet
+
+build/*
```

Comment:
@thomaseizinger @bonomat I'm not too happy with what I've done here, but otherwise the `.gitignore` file inside `waves/build` is being deleted every time we build the frontend. Other ideas are welcome :D

Reply:
Why are you not happy with this? Not versioning the build output seems fine to me?

Reply:
Sorry, I meant to link to the whole commit https://github.com/comit-network/droplet/pull/80/commits/5743824719fbcca2b97e12a5792af94ef9cc5776. I had to modify CI after this change.